README.md

```diff
@@ -154,7 +194,11 @@ How many valid sentences are there in this language?
 
 ## A2.6
 
-«replace this with your answer»
+Lazy and/or smelly create 5 possible combinations. Dog or cat is 2 combinations.
+<br/>
+Ate or run is 2 combinations, and slowly or noisily is 3 combinations because it
+<br/>
+could be absent. So, 5*2*2*3 = 60 different valid sentences.
```

Comment:
# -1

In the original grammar there are an infinite number (because the adjectives can repeat indefinitely. However in your grammar, your answer is true).